dist/screenfull.js

```diff
@@ -83,7 +82,13 @@
 	};
 
 	var screenfull = {
-		request: function (elem) {
+		request: function (options) {
+			var defaults = {
+				element: document.documentElement,
+				fullscreenOptions: {navigationUI: 'auto'}
+			};
+			options = Object.assign({}, defaults, options);
```

Comment:
You need to add to the `Support` section in the readme that IE users needs the https://github.com/sindresorhus/object-assign ponyfill.